hs_core/models.py

```diff
@@ -3498,6 +3498,7 @@ def get_hs_term_dict(self):
 
         hs_term_dict["HS_RES_ID"] = self.short_id
         hs_term_dict["HS_RES_TYPE"] = self.resource_type
+        hs_term_dict.update(self.extra_metadata.items())
```

Comment:
@hyi - here is the functional change.   It's small and easy to miss.

Reply:
@sblack-usu Got it. I actually did not miss this line, but just did not relate this to the functional change. Now it makes sense to me. I'll approve the PR.